symengine/cwrapper.cpp

```diff
@@ -41,6 +41,8 @@ extern "C" {
 
 struct CRCPBasic {
     SymEngine::RCP<const SymEngine::Basic> m;
+    CRCPBasic() {};
```

Comment:
I see. Let me check.

*Regards,*

*Abinash Meher*

On Sat, Jan 30, 2016 at 1:50 PM, Isuru Fernando <notifications@github.com>
wrote:

> In symengine/cwrapper.cpp
> <https://github.com/symengine/symengine/pull/781#discussion_r51342324>:
>
> > @@ -41,6 +41,8 @@ extern "C" {
> >
> >  struct CRCPBasic {
> >      SymEngine::RCP<const SymEngine::Basic> m;
> > +    CRCPBasic() {};
>
> You don't need this anymore. I think this is what is causing the error,
> because an RCP object can't be null in Teuchos
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/symengine/symengine/pull/781/files#r51342324>.
>